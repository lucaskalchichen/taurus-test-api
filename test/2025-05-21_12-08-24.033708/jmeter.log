2025-05-21 12:08:33,584 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 12:08:33,595 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 12:08:33,595 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 12:08:33,595 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708/jmeter-bzt.properties
2025-05-21 12:08:33,597 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708/system.properties
2025-05-21 12:08:33,601 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 12:08:33,601 INFO o.a.j.JMeter: Version 5.5
2025-05-21 12:08:33,601 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 12:08:33,601 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 12:08:33,601 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 12:08:33,601 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 12:08:33,602 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 12:08:33,602 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 12:08:33,602 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 12:08:33,602 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 12:08:33,602 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 12:08:33,605 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 12:08:33,605 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 12:08:33,605 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 12:08:33,605 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 12:08:33,606 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 12:08:33,611 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 12:08:33,611 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708
2025-05-21 12:08:33,612 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708
2025-05-21 12:08:33,613 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708
2025-05-21 12:08:33,619 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 12:08:33,619 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708'
2025-05-21 12:08:33,693 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 12:08:33,704 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 12:08:33,705 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 12:08:33,707 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708/modified_requests.jmx
2025-05-21 12:08:33,723 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 12:08:33,724 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 12:08:33,731 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:08:33,731 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:08:33,731 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:08:33,731 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:08:33,731 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 12:08:33,732 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 12:08:33,748 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 12:08:33,749 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 12:08:33,749 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 12:08:33,754 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 12:08:33,755 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 12:08:33,885 INFO o.a.j.JMeter: Running test (1747840113885)
2025-05-21 12:08:33,895 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 12:08:33,895 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 12:08:33,896 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 12:08:33,896 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 12:08:33,901 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 12:08:33,913 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 12:08:33,914 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 12:08:33,914 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 12:08:33,916 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 12:08:33,917 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 12:08:33,917 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 12:08:33,917 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 12:08:33,917 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 12:08:33,971 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 12:08:34,040 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 12:08:34,111 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 12:08:34,183 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 12:08:34,253 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 12:08:34,323 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 12:08:34,393 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 12:08:34,465 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 12:08:34,536 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 12:08:34,621 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 12:08:34,626 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 12:08:34,626 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 12:08:34,678 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 12:08:34,750 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 12:08:34,820 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 12:08:34,891 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 12:08:34,961 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 12:08:35,044 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 12:08:35,046 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 12:08:35,047 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 12:08:35,104 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 12:08:35,176 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 12:08:35,246 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 12:08:35,317 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 12:08:35,388 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 12:08:35,458 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 12:08:35,515 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 12:08:35,516 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 12:08:35,530 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 12:08:35,601 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 12:08:35,672 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 12:08:35,742 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 12:08:35,814 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 12:08:35,886 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 12:08:35,957 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 12:08:35,995 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 12:08:35,997 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 12:08:36,027 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 12:08:36,097 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 12:08:36,169 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 12:08:36,239 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 12:08:36,310 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 12:08:36,382 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 12:08:36,452 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 12:08:36,454 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 12:08:36,454 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 12:08:36,523 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 12:08:36,594 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 12:08:36,666 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 12:08:36,736 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 12:08:36,808 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 12:08:36,974 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 12:08:36,975 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 12:08:37,485 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 12:08:37,486 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 12:08:37,961 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 12:08:37,962 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 12:08:38,445 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 12:08:38,445 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 12:08:38,841 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 12:08:38,842 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 12:08:39,243 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 12:08:39,244 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 12:08:39,716 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 12:08:39,716 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 12:08:40,079 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 12:08:40,079 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 12:08:40,460 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 12:08:40,460 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 12:08:40,906 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 12:08:40,906 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 12:08:41,377 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 12:08:41,377 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 12:08:41,827 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 12:08:41,827 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 12:08:42,277 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 12:08:42,277 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 12:08:42,734 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 12:08:42,734 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 12:08:43,197 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 12:08:43,197 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 12:08:43,643 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 12:08:43,643 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 12:08:44,093 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 12:08:44,093 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 12:08:44,552 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 12:08:44,553 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 12:08:44,997 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 12:08:44,997 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 12:08:45,456 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 12:08:45,457 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 12:08:45,904 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 12:08:45,905 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 12:08:46,362 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 12:08:46,362 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 12:08:46,819 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 12:08:46,819 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 12:08:47,281 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 12:08:47,281 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 12:08:47,730 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 12:08:47,730 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 12:08:48,196 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 12:08:48,197 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 12:08:48,630 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 12:08:48,631 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 12:08:49,094 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 12:08:49,095 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 12:08:49,542 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 12:08:49,542 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 12:08:50,010 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 12:08:50,010 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 12:08:50,442 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 12:08:50,443 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 12:08:50,814 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 12:08:50,814 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 12:08:51,261 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 12:08:51,261 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 12:08:51,699 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 12:08:51,699 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 12:08:52,137 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 12:08:52,137 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 12:08:52,623 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 12:08:52,623 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 12:08:53,017 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 12:08:53,017 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 12:08:53,018 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 12:08:53,019 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
