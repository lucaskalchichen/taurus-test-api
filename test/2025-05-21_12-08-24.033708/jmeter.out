Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-08-24.033708/modified_requests.jmx
Starting standalone test @ 2025 May 21 12:08:33 ART (1747840113744)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 12:08:53 ART (1747840133019)
... end of run
